src/witan/models/load_data.clj

```diff
@@ -45,7 +45,7 @@
   (make-ordered-ds-schema [[:gss-code s/Str] [:district s/Str] [:sex s/Str] [:age s/Int]
                            [:var s/Str] [:year s/Int] [:estimate s/Num]]))
 
-(def HistPopnEst
+(def BasePopn
```

Comment:
I think the less repetition of code, the better. Schemas are best in their own NS, usually